BuildLoader.py

```diff
@@ -1446,6 +1484,12 @@ def post_build(self):
         if getattr(self._board, "GetContainerList", None):
             container_list = self._board.GetContainerList ()
             component_dir = os.path.join(os.environ['PLT_SOURCE'], 'Platform', self._board.BOARD_PKG_NAME, 'Binaries')
+            device_info_file = os.path.join(component_dir, 'DeviceInfo.bin')
+            if os.path.exists(device_info_file):
+                print(f"Found DeviceInfo.bin at {device_info_file}, adding to container list")
+                self.add_device_info_container(container_list, device_info_file)
+            else:
+                print(f"DeviceInfo.bin not found at {device_info_file}, skipping container list addition")
```

Comment:
Avoid updating common build tool for a specific component if possible. How about just updating BoardConfig script adding TMAC for MAC address?